find_green_build

```diff
@@ -122,7 +125,8 @@ fi
 
 logecho
 release::set_build_version $TESTING_BRANCH $LOCAL_CACHE \
-                           "$FLAGS_exclude_suites" || common::exit 1
+                           "$FLAGS_exclude_suites" \
+                           $FLAGS_limit || common::exit 1
```

Comment:
Could we move the default value here instead of in `set_build_version`? Currently if someone adds another argument, they need to know/remember to account for `hard_limit` being optional.